jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenHelpfulSuggestions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven;
+
+import com.google.cloud.tools.jib.plugins.common.HelpfulSuggestions;
+
+/** Builder for Maven-specific {@link HelpfulSuggestions}. */
+class MavenHelpfulSuggestions extends HelpfulSuggestions {
+
+  MavenHelpfulSuggestions(String messagePrefix) {
+    super(messagePrefix, "mvn clean", "<to><image>", "-Dimage", "pom.xml");
```

Comment:
note: this class is renamed from `MavenHelpfulSuggestionsBuilder` below after simplified.